pystorm/component.py

```diff
@@ -36,6 +39,8 @@
                       'debug': logging.DEBUG,
                       'trace': logging.DEBUG}
 _SERIALIZERS = {"json": JSONSerializer, "msgpack": MsgpackSerializer}
+# Used for generating namedtuple type names
```

Comment:
Let's clarify this comment:

    # Convert names to valid Python identifiers, replace whitespace and leading digits.